modules/ROOT/pages/ibm/ibm-ctg-connector.adoc

```diff
@@ -327,32 +345,32 @@ output application/json
 [source,json,linenums]
 ----
 {
-    date: "08/01/18 13:41:17"
+    date: "08/01/2019 13:41:17"
 }
 ----
 
 === Invoke a Channel Program
 
 This application calls EC03, a channel-based program that takes an input data container and returns three containers:
 
-* A data/time container.
+* A data and time container.
 * The length of the input data and returns uses channels and containers in a CICS program.
```

Comment:
Please review this list item, which appears improperly worded. I am not sure why "returns" is repeated or how the length is a return value....